source/fundamentals/connections/connection-options.txt

```diff
@@ -0,0 +1,201 @@
+   * - **heartbeatFrequencyMS**
+     - integer
+     - ``10000``
+     - Specifies the number of milliseconds to wait between periodic
+       background server checks.
+
+   * - **tls**
+     - boolean
+     - ``false``
+     - Specifies whether to establish a Transport Layer Security (TLS)
+       connection with the instance. This is automatically set to ``true``
+       when using a DNS seedlist (SRV) in the connection string. You can
+       override this behavior by setting the value to ``false``.
+
+   * - **w**
+     - string or integer
+     - ``null``
+     - Specifies the write concern. To learn more about values, see the
+       server documentation on
+       :manual:`Write Concern options </reference/write-concern>`.
+
+   * - **directConnection**
+     - boolean
+     - ``false``
+     - Specifies whether to force dispatch **all** operations to the host
+       specified in the connection URI.
+
+For a full list of connection options, see the `ClientOptions API
+documentation
+<{+api+}/mongo/options#ClientOptions>`__.
+
+.. _golang-timeout-setting:
+
+Single Timeout Setting
+----------------------
+
+You can set a single ``Timeout`` option on your ``Client`` instance to
+specify the maximum amount of time that a single operation can take to
+execute.
+
+Set a timeout by calling the ``SetTimeout()`` method when specifying
+options for your ``Client`` instance or by specifying the
+``timeoutMS`` option in your connection URI. By default, all
+``Database``, ``Collection``, ``Session``, ``ChangeStream``, and
+``Bucket`` instances elsewhere in your application inherit the
+``Timeout`` option from ``Client`` if you do not set a different timeout
+on specific operations in the operation's Context.
+
+If you set a timeout on a Context passed into an operation, the driver uses
+that value for the operation. If you do not specify a Context timeout,
+the operation Context derives the timeout from the ``Client`` instance.
+
+.. note:: Retries under Timeout Specification
+
+   When using default settings, if you set a ``Timeout`` option on your ``Client``
+   and your operation requires a retry, the driver retries the operation
+   as many times as possible before the timeout expires. Once the timeout
+   expires, the driver returns a timeout error. Versions 1.1 and later
+   of the {+driver-short+} enable retryable reads and writes by default.
+   See the Server manual for more information about :ref:`retryable
+   reads <retryable-reads>` and :manual:`retryable writes </core/retryable-writes/>`.
+
+Timeout Examples
+~~~~~~~~~~~~~~~~
+
+This section provides examples that demonstrate different ways to set the
+``Timeout`` setting in your application.
+
+Client Option
+^^^^^^^^^^^^^
+
+The following code shows how to set the ``Timeout`` option on a ``Client``
+by using the ``SetTimeout()`` method:
+
+.. code-block:: go
+
+   opts := options.Client().SetTimeout(5 * time.Second)
+   client, err := mongo.Connect(opts)
+
+Connection String Option
```

Comment:
Suggest specifying that the `timeoutMS` query parameter is a client-level timeout.